lib/ansible/modules/cloud/huawei/hwc_network_vpc.py

```diff
@@ -0,0 +1,513 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2018 Huawei
+# GNU General Public License v3.0+ (see COPYING or
+# https://www.gnu.org/licenses/gpl-3.0.txt)
+# ----------------------------------------------------------------------------
```

Comment:
This might be auto generated, but if pushing into this repo, you must accept changes done directly to these files.
 
PRs that override local changes w/o a specific reason are unacceptable.

Reply:
Done